welcome/revhistory_full.adoc

```diff
@@ -11,13 +11,14 @@ date.
 
 // do-release: revhist-tables
 == Mon Aug 15 2016
-.Install Config
+.Installation and Configuration
 include::install_config/revhistory_install_config.adoc[tag=install_config_mon_aug_15_2016]
+
 == Mon Aug 08 2016
-.Install Config
+.Installation and Configuration
 include::install_config/revhistory_install_config.adoc[tag=install_config_mon_aug_08_2016]
 
-.Dev Guide
+.Developer Guide
```

Comment:
Also taking this opportunity to spell out some recent shortened book titles here.

@ahardin-rh @bfallonf @tpoitras @vikram-redhat ^ Reminder to try and manually spell all book titles out fully in the `welcome/revhistory_full` file after the release script is finished.